components/layout/display_list_builder.rs

```diff
@@ -196,16 +198,31 @@ impl<'a> DisplayListBuildState<'a> {
         }
     }
 
+    fn update_overflow_for_stacking_context(&mut self, stacking_context: &mut StackingContext) {
+        if stacking_context.context_type != StackingContextType::Real {
+            return;
+        }
+
+        for child in self.stacking_context_children(stacking_context.id).iter_mut() {
```

Comment:
Nice catch! I've removed ```stacking_context_children(...)``` and instead just handle the case where there is no entry. The resulting version of ```update_overflow_for_stacking_context``` isn't particularly pretty, but I'm going to remove that method entirely in a followup PR.